Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -6198,3 +6199,20 @@ void MainWindow::slotEnableVoiceActivation(bool checked)
 {
     slotMeEnableVoiceActivation(checked, SOUNDEVENT_VOICEACTMEON, SOUNDEVENT_VOICEACTMEOFF);
 }
+
+bool MainWindow::eventFilter(QObject *object, QEvent *event)

```

Comment:
Can't the event filter be implemented in ChannelsTree-class? It would be more correct to place it there.

Reply:
I don't succeed to place it in ChannelsTree class, probably because there is no ui element to place it. My tests has finished by a crash when pressing any key on channels tree :(